### PR TITLE
Use docker migration script instead in cargo make

### PR DESCRIPTION
**Pull request by TimDiekmann**
## 🌟 What is the purpose of this PR?

`cargo-make` is not able to resolve environment variables in the `env_files` attribute. In order to make the `.env` file-path absolute, we use the script defined in `deployment/migrations/Dockerfile` instead.

## 🔗 Related links

- https://github.com/sagiegurari/cargo-make/issues/723

## ⚠️ Known issues

Instead of relying on `docker-compose` we could also use the `Dockerfile` directly, but this requires that it needs to be built in a seperated step. The `--no-recreate` parameter avoids restarting the Postgres deployment if it's already running, but if it's not running, it will start it as well.

An alternative solution is to change `env-files` to `[".env"]`, which may break when running from other directories.

### CODE REVIEW

File: packages/graph/Makefile.toml [hunk 1]
```diff
@@ -23,10 +23,9 @@ cwd = "${REPO_ROOT}/packages/graph/migrations"
 [tasks.migrate-up]
 category = "Deploy"
 description = "Runs all migrations"
-extend = "yarn"
-args = ["graph:migrate", "up"]
```
Comment (Alfred-Mountfield) [inline packages/graph/Makefile.toml:27]: Is there even a purpose of us keeping this `graph:migrate` command in yarn then?

I know @benwerner01 found it somewhat confusing that the yarn had two different migration commands that behave differently. (And I mean I think we should seek to unify them at some point anyway, but in the meantime perhaps it'd be best to remove one of the `yarn` ones and leave this as a manual docker thing expressed through Cargo make, Rust tooling for Rust workflows)

Comment (TimDiekmann) [reply packages/graph/Makefile.toml:27]: The Dockerfile will use the `yarn` command, so fully remove it isn't possible. An alternative would be to simply remove the `cargo make` commands for `migrate-up` and `recreate-db` and run the yarn commands directly, however, we need to pass the .env file path manually then.

The easiest way is probably to just keep them for now.

**Review by Alfred-Mountfield — DISMISSED** [review]
I think this is fine as a _temporary_ fix. But we know we need to handle and improve the development experience in general here, especially with regards to starting up dependencies.

**Review by Alfred-Mountfield — APPROVED** [review]

**Review by Alfred-Mountfield — CHANGES_REQUESTED** [review]
External services failed to start. Request my review again when it passes :)

**Comment by TimDiekmann** [comment]
> External services failed to start. Request my review again when it passes :)

CI is green now.

**Review by Alfred-Mountfield — APPROVED** [review]
🚀

**Pull request merged**